client/css/style.css

```diff
@@ -818,6 +819,11 @@ button,
 	color: #2ecc40;
 }
 
+#chat .ctcp .from:before {
+	font-family: FontAwesome;
+	content: "\f0f6";
+}
+
```

Comment:
It actually can't as current lounge handles CTCP requires as mere notices, and irc-fw has a separate callback for it.